routes/routeStack.js

```diff
@@ -1,32 +1,77 @@
-import { createStackNavigator } from "react-navigation-stack";
+import { createStackNavigator, TransitionPresets } from "react-navigation-stack";
 import { createAppContainer } from "react-navigation";
 import HomeMap from "../screens/Map";
-import Directions from "../components/Directions";
 import MoreDetails from "../screens/MoreDetails";
 import DoubleSearch from "../screens/DoubleSearch";
+import MapDirections from "../screens/Directions";
+import PreviewDirections from "../screens/PreviewDirections";
+import PreferenceMenu from "../screens/PreferenceMenu";
+import IndoorMapView from "../screens/Indoor/IndoorMapView";
+import Nearby from "../screens/Nearby";
+
 // Add screens here
 const screens = {
```

Comment:
Implementing navigation has made the application faster and the transitions smoother, good job!

Reply:
💯 🥇

Reply:
👍